index.d.ts

```diff
@@ -0,0 +1,5 @@
+declare namespace NodeJS {
+  export interface Process {
+    browser: boolean
+  }
+}
```

Comment:
Move this and the other type def into a `types` directory, just to keep things clean and consistent.